### PR TITLE
🐛 🎨 Icons are not properly rendering-made change to render properly

### PR DESCRIPTION
**Pull request by Nyash-Mauro**
# Description

Some Icons were missing i.e the search and bot menu they were not being rendered correctly. I added material icons to generate the icons and be displayed them correctly.

ref #115 
This change fixes issue ref#115

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
icons-shwing

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Nyash-Mauro** [comment]
search
This is the final change

**Comment by Nyash-Mauro** [comment]
final-icons
THis is the site after removal of the font awesome icon that was not being used and doesn't exist in the design soecification

File: libs/features/convs-mgr/home/src/lib/components/story-list-item/story-list-item.component.html [hunk 1]
```diff
@@ -9,9 +9,10 @@ <h2 class="bot">
         </h2>
       </div>
 
-      <div class="dots">
-          <i class="fas dotedIcon"></i>
-      </div>
+      <button mat-icon-button>
+        <mat-icon>more_vert</mat-icon>
+      </button>
```
Comment (IanOdhiambo9) [inline libs/features/convs-mgr/home/src/lib/components/story-list-item/story-list-item.component.html:14]: Please change this icon to: "i class="fas fa-ellipsis-v""

File: libs/features/convs-mgr/home/src/lib/components/story-list/story-list.component.html [hunk 2]
```diff
@@ -1,13 +1,11 @@
 <!-- edited the search -->
 <div class="search">
-  <i class="fas searchIcon" ></i>
+  <i class="fas searchIcon" >        
+    <mat-icon>search</mat-icon>
+  </i>
```
Comment (IanOdhiambo9) [inline libs/features/convs-mgr/home/src/lib/components/story-list/story-list.component.html:5]: Just use i class="fas fa-search" and flip with CSS to match design and color

File: libs/features/convs-mgr/home/src/lib/modals/create-bot-modal/create-bot-modal.component.ts [hunk 3]
```diff
@@ -15,7 +15,6 @@ export class CreateBotModalComponent implements OnInit {
   createFormGroup(){
     this.botForm = this._formBuilder.group({
       botName: [this._addStory$.generateName()],
-      // botImage: [this._addImage$.uploadFile(File)],
       botDesc: ['']
```
Comment (IanOdhiambo9) [inline libs/features/convs-mgr/home/src/lib/modals/create-bot-modal/create-bot-modal.component.ts:18]: why is this removed?

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

**Review by IanOdhiambo9 — APPROVED** [review]

**Pull request merged**